go.mod

```diff
@@ -150,4 +152,12 @@ replace (
 	k8s.io/cli-runtime => k8s.io/cli-runtime v0.25.4
 	k8s.io/client-go => k8s.io/client-go v0.25.4
 	k8s.io/code-generator => k8s.io/code-generator v0.25.4
+
+	//Pin Knative versions
```

Comment:
Adding all packages for the convenience to avoid newer version from plugins afterwards.